examples/PlotLFRE.ipynb

```diff
@@ -222,7 +141,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.6.11"
+   "version": "3.7.6"
```

Comment:
I guess this is a change of the newer notebook version I am using. I am not sure if I should just remove it, or change it to something consistent. Do you have any thoughts?

Reply:
You should remove these sorts of edits from the notebooks before merging. Do you generally make sure to strip the NBs?